### PR TITLE
CLOUDP-94137: Capability to shutdown/startup the entire cluster already exists and we’ll be adding the capabilities to shutdown/startup a set of processes

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-94137

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

Screenshot 2021-07-08 at 16 40 02

```
➜  ~/workspace/mongoCLI git:(CLOUDP-94137) ✗ ./bin/mongocli om cluster shutdown  myReplicaSet --processName andreaMac:27078 -P ops
? Are you sure you want to shutdown: myReplicaSet (andreaMac:27078) Yes

➜  ~/workspace/mongoCLI git:(CLOUDP-94137) ✗ ./bin/mongocli om cluster shutdown  ClusterAndrea --processName andreaMac:26120 -P ops
? Are you sure you want to shutdown: ClusterAndrea (andreaMac:26120) Yes
Changes are being applied, please check http://127.0.0.1:8080/v2/60e58432700ada55e2660a37#deployment/topology for status

➜  ~/workspace/mongoCLI git:(CLOUDP-94137) ✗ ./bin/mongocli om cluster startup myReplicaSet --processName andreaMac:27078 -P ops
? Are you sure you want to startup: myReplicaSet (andreaMac:27078) Yes
Changes are being applied, please check http://127.0.0.1:8080/v2/60e58432700ada55e2660a37#deployment/topology for status

➜  ~/workspace/mongoCLI git:(CLOUDP-94137) ✗ ./bin/mongocli om cluster startup  ClusterAndrea --processName andreaMac:26120 -P ops
? Are you sure you want to startup: ClusterAndrea (andreaMac:26120) Yes
Changes are being applied, please check http://127.0.0.1:8080/v2/60e58432700ada55e2660a37#deployment/topology for status

./bin/mongocli om cluster shutdown ClusterAndrea --processName andreaMac:26120,andreaMac:26119 -P ops
? Are you sure you want to shutdown: ClusterAndrea (andreaMac:26120, andreaMac:26119) Yes
Changes are being applied, please check http://127.0.0.1:8080/v2/60e58432700ada55e2660a37#deployment/topology for status
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
Waiting for https://github.com/mongodb/go-client-mongodb-ops-manager/pull/111

File: internal/cli/opsmanager/clusters/shutdown.go [hunk 1]
```diff
@@ -71,24 +76,42 @@ func (opts *ShutdownOpts) Confirm() error {
...
+	}
 	prompt := &survey.Confirm{
-		Message: fmt.Sprintf("Are you sure you want to shutdown: %s", opts.name),
+		Message: fmt.Sprintf("Are you sure you want to shutdown: %s", shutdownProcess),
 	}
 	return survey.AskOne(prompt, &opts.confirm)
 }
 
-// mongocli cloud-manager cluster(s) shutdown <name> --projectId projectId [--force].
+func (opts *ShutdownOpts) validateInputs() error {
```
Comment (gssbzn) [inline internal/cli/opsmanager/clusters/shutdown.go:91]: do you need this? cluster name is always present and validated and processes are optional

Comment (andreaangiolillo) [reply internal/cli/opsmanager/clusters/shutdown.go:91]: ops, left over

File: internal/cli/opsmanager/clusters/startup.go [hunk 2]
```diff
@@ -71,31 +76,51 @@ func (opts *StartupOpts) Confirm() error {
...
+
 	prompt := &survey.Confirm{
-		Message: fmt.Sprintf("Are you sure you want to startup: %s", opts.name),
+		Message: fmt.Sprintf("Are you sure you want to startup: %s", startupProcess),
 	}
 	return survey.AskOne(prompt, &opts.confirm)
 }
 
-// mongocli cloud-manager cluster(s) startup <name> --projectId projectId [--force].
+func (opts *StartupOpts) validateInputs() error {
```
Comment (gssbzn) [inline internal/cli/opsmanager/clusters/startup.go:92]: [q] same question

**Review by gssbzn — COMMENTED** [review]
looks good with a somewhat blocking question, `validateInputs` I don't think you need that

**Review by gssbzn — APPROVED** [review]
LGTM let's schedule the cloud manager and ops manager cluster tests good to merge if they pass

**Pull request merged**